subxt/src/utils/unchecked_extrinsic.rs

```diff
@@ -0,0 +1,91 @@
+// Copyright 2019-2023 Parity Technologies (UK) Ltd.
+// This file is dual-licensed as Apache-2.0 or GPL-3.0.
+// see LICENSE for license details.
+
+//! The "default" Substrate/Polkadot UncheckedExtrinsic.
+//! This is used in codegen for runtime API calls.
+//!
+//! The inner bytes represent the encoded extrinsic expected by the
+//! runtime APIs. Deriving `EncodeAsType` would lead to the inner
+//! bytes to be re-encoded (length prefixed).
+
+use std::marker::PhantomData;
+
+use codec::Decode;
+use scale_decode::{visitor::DecodeAsTypeResult, IntoVisitor, Visitor};
+
+/// The unchecked extrinsic from substrate.
+#[derive(Clone, Debug, Eq, PartialEq, Decode)]
+pub struct UncheckedExtrinsic<Address, Call, Signature, Extra>(
+    pub Vec<u8>,
+    #[codec(skip)] pub PhantomData<(Address, Call, Signature, Extra)>,
+);
+
+impl<Address, Call, Signature, Extra> codec::Encode
+    for UncheckedExtrinsic<Address, Call, Signature, Extra>
+{
+    fn encode(&self) -> Vec<u8> {
+        self.0.to_owned()
+    }
```

Comment:
Nit: Better to impl `encode_to` I think, which is given a buffer to write to; this will save cloning the vec in the event that the caller already has a buffer :)

Reply:
(I note that `Encoded` impls `encode` and not `encode_to` so that could be improved)